source/_integrations/amcrest.markdown

```diff
@@ -167,7 +40,9 @@ Newer Amcrest firmware may not work, then `rtsp` is recommended instead.
 make sure to follow the steps mentioned at [FFmpeg](/integrations/ffmpeg/)
```

Comment:
Does this still apply even? It can be a dependency, right?

Reply:
I tried running with just the default config and an Amcrest camera with rtsp, and it looks like everything is able to run just fine with that, so I can remove this.